src/tidecli/models/task_data.py

```diff
@@ -28,6 +29,15 @@ class TaskFile(BaseModel):
     source: str = "editor"
     """Source attribute in TIM. Not used in CLI app."""
 
+    task_directory: str | None = None
+    """Directory of the task."""
+
+    task_type: str | None = None
+    """Type of the task."""
+
+    saved_full_path: str | None = None
```

Comment:
I'm not sure I understand the meaning of saved_full_path, what's the difference between path and full path? Would not be simply `path`, `save_path`, or `absolute_path` (if the path is meant to be absolute path) be easier to understand?

Reply:
path is used in task (TaskData) level to mean Tasks tim directory (the original Tide name thaou
could have been better to name f.ex TasksPathInTim).  Now its is not good idea to use `path`  in
different meaning in TaskFile. Closest in TaskFile is `file_name`, but it's harder to compare
the absolute path of the file that is meant to be submitted.  So its is the absolute name of the file saved to the disk.  Maybe saved_absolute_name  is better?  path is always difficult because it may mean directory part or full name.

Reply:
path doesn't have to be difficult or ambiguous. Within one project we can decide what is the meaning and stick to it. Then add describing prefixes, suffixes when meanings deviate from the "default".

I would go for either absolute_path, or absolute_save_path if we need the "save" to differentiate from other uses of path within tide-cli code.

Reply:
Now the name is "saved_absolute_file_name" but as I said earlier, the code owner can refactor the
names to follow some guideline when he has better overall perspective.

And when doing that there is option that the logic is changes so that instead of absolute, it might be
relative anyway so that the whole directory could be moved another location.  This does not affect to the interface.

I used name "saved", because at the moment it is a mess how the name is "counted".  And then to me
it is more clear that what ever the attributes says, the file is saved to this name and could be found by that name.